src/fil_cns/validation.rs

```diff
@@ -10,6 +10,7 @@ use crate::chain_sync::collect_errs;
 use crate::metrics::HistogramTimerExt;
 use crate::networks::{ChainConfig, Height};
 use crate::shim::actors::PowerActorStateLoad as _;
+use crate::shim::actors_interface::power;
```

Comment:
`_iterface` looks redundant when it's under `shim`, what do you think?

Reply:
done